app/controllers/api/v1/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class SessionsController < ApplicationController
+      before_action :authenticate
+
+      # TODO: samuel - Bypass CSRF token for now
+      protect_from_forgery with: :exception
+
+      def create
+        user = User.find_by(email: params[:user][:email])
+
+        if user.present? && user.authenticate(params[:user][:password])
+          session[:user_id] = user.id
+          render json: { status: :success, signed_in: true }, status: :no_content
+        else
+          render json: { status: :error, signed_in: false }, status: :bad_request
```

Comment:
I'm not sure what you mean.
This should only cover the Authentication error?

Reply:
No, this is how it should be IMHO!
A request with a non present user in the database should still be considered as a bad request to avoid providing further data that may be used as an attack vector.
 I like and encourage what you have done,